server/@types/ui/index.d.ts

```diff
@@ -275,3 +275,26 @@ export type PartnerAgencyDetails = {
   phoneNumber: string
   roleInPlan: string
 }
+
+export type ContingencyPlanQuestionId =
+  | 'noReturn'
+  | 'placementWithdrawn'
+  | 'victimConsiderations'
+  | 'unsuitableAddresses'
+  | 'suitableAddresses'
+  | 'breachInformation'
+  | 'otherConsiderations'
+
+type ContingencyPlanQuestionsBody = Record<ContingencyPlanQuestionId, string>
+
+type ContingencyPlanQuestion = {
+  question: string
+  hint?: string
+  error: string
+}
+
+export type ContingencyPlanQuestionsRecord = Record<ContingencyPlanQuestionId, ContingencyPlanQuestion>
+
+export type ContingencyPlanQuestionsBody = {
+  [key in BodyProperties]: string
+}
```

Comment:
I'm unsure what's going on here, in combination with `type ContingencyPlanQuestionsBody = Record<ContingencyPlanQuestionId, string>` above

Reply:
Yeah this is a duplicate type, I think `Record<ContingencyPlanQuestionId, string>` is more readable so kept that version

Reply:
Cool. I think this was confusing me because I couldn't see where `BodyProperties` was coming from